aries-site/src/data/wcag/components.json

```diff
@@ -48,6 +48,26 @@
       { "rule": "3.2.5", "status": "passed" },
       { "rule": "4.1.2", "status": "passed" }
     ],
+      "pageheader": [
+    { "rule": "1.1.1", "status": "conditional" },
+    { "rule": "1.3.1", "status": "passed" },
+    { "rule": "1.3.4", "status": "passed" },
+    { "rule": "1.4.3", "status": "passed" },
+    { "rule": "1.4.6", "status": "failed" },
+    { "rule": "2.1.1", "status": "passed" },
+    { "rule": "2.1.3", "status": "passed" },
+    { "rule": "2.4.3", "status": "passed" },
+    { "rule": "2.4.6", "status": "passed" },
+    { "rule": "2.4.11", "status": "passed" },
+    { "rule": "2.4.12", "status": "passed" },
+    { "rule": "2.4.13", "status": "passed" },
+    { "rule": "2.5.5", "status": "failed" },
+    { "rule": "2.5.8", "status": "passed" },
+    { "rule": "3.2.3", "status": "passed" },
+    { "rule": "3.3.2", "status": "passed" },
```

Comment:
I was thinking rule 3.3.2 would be not-applicable. Could you elaborate more on your thinking here?